packages/scripts/README.md

```diff
@@ -10,6 +10,28 @@ See https://6.docs.plone.org/volto/development/i18n.html for more information.
 
 This script is installed in the `node_modules/.bin` directory and can be called via `yarn i18n` or directly in the `scripts` `package.json` part.
 
+## `volto-update-deps`
+
+Given an existing project path, `volto-update-deps` updates the Volto `dependencies` and `devDependencies` for the existing Volto (`@plone/volto`) version in the `package.json` file.
+
+If you are in the process of upgrading your project to a given Volto version, you should update the `package.json` `@plone/volto` dependencies key to use the updated version, then run the script.
+The script is included in the `@plone/scripts` package.
+You can run it from inside your project's directory root.
+
+```shell
+pnpm exec volto-update-deps .
+```
+
+Or run it directly from the `.bin` folder.
+
+```shell
+./node_modules/.bin/volto-update-deps.js path/to/my/project
+```
+
+## Deprecated utilites
+
+The following utilities are no longer used by core or in projects.
+
```

Comment:
All of the following utilities' headings should be increased to be subsections of this section.

Reply:
Done